src/http/common.cr

```diff
@@ -46,7 +46,7 @@ module HTTP
     bytesize = line.bytesize
 
     # Get the colon index and name
-    colon_index = cstr.to_slice(bytesize).index(':'.ord) || 0
+    colon_index = cstr.to_slice(bytesize).index(':'.ord.to_u8) || 0
```

Comment:
@kostya Not a bug because `index` checks for equality and it works with different kinds of integers. With this change I have to put the specific type, but I think these conversions will be rare, so it's ok.